src/Pages/Overview/Overview.js

```diff
@@ -85,6 +95,24 @@ export default function Overview(props) {
     );
   }, [page, rowsPerPage, users, total]);
 
+  function handleSortUsers(columnName) {
+    if (columnName == null) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness
```suggestion
    if (columnName === null) {
```